templates/layout.jade

```diff
@@ -31,3 +31,11 @@ html(lang='en')
             p &copy; #{ new Date().getFullYear() } #{ locals.owner } &mdash; powered by&nbsp;
               a(href='https://github.com/jnordberg/wintersmith') Wintersmith
               //- please leave the "powered by" if you use the design
+    //- Google analytics
+    script
+     | (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+     | (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+     | m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+     | })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+     | ga('create', 'UA-49880770-1', 'auto');
+     | ga('send', 'pageview');
```

Comment:
When I do development, I typically use LiveReload which will bloat any production metrics. Can we either:

- Move this behind a `NODE_ENV` variable so that in development it isn't built but on release it is?
    - Con: No way to test in development without affecting production metrics

```jade
if process.env.NODE_ENV == 'production'
  script
    | (function(i,s,o,...
```